.gitignore

```diff
@@ -238,3 +238,4 @@ ModelManifest.xml
 *.pdf
 *.run.xml
 *.synctex.gz
+.axoCover
```

Comment:
The diff seems off... This is already on the master branch. https://github.com/smith-chem-wisc/ProteoformSuite/blob/master/.gitignore#L241